README.md

```diff
@@ -20,39 +20,39 @@ For details of the other Kata Containers repositories, see the
 
 ## Getting Started
 
-* [Installation guides](./install/README.md): Install and run Kata Containers with Docker or Kubernetes
+* [Installation guides](./install/README.md): Install and run Kata Containers with Docker\* or Kubernetes\*
 
 ## More User Guides
 
-* [Upgrading](Upgrading.md): how to upgrade from [Clear Containers](https://github.com/clearcontainers) and [runV](https://github.com/hyperhq/runv) to [Kata Containers](https://github.com/kata-containers) and how to upgrade an existing Kata Containers system to the latest version.
-* [Limitations](Limitations.md): differences and limitations compared with the default [Docker](https://www.docker.com/) runtime,
+* [Upgrading](Upgrading.md): How to upgrade from [Intel® Clear Containers](https://github.com/clearcontainers) and [runV](https://github.com/hyperhq/runv) to [Kata Containers](https://github.com/kata-containers) and how to upgrade an existing Kata Containers system to the latest version.
+* [Limitations](Limitations.md): Differences and limitations compared with the default [Docker](https://www.docker.com/) runtime,
 [`runc`](https://github.com/opencontainers/runc).
 
 ### Howto guides
 
-See the [howto documentation](how-to).
+See the [how-to documentation](how-to).
 
-## Kata Use-Cases
+## Kata Container Use-Cases
 
-* [GPU Passthrough with Kata](./use-cases/GPU-passthrough-and-Kata.md)
-* [OpenStack Zun with Kata Containers](./use-cases/zun_kata.md)
-* [SR-IOV with Kata](./use-cases/using-SRIOV-and-kata.md)
-* [Intel QAT with Kata](./use-cases/using-Intel-QAT-and-kata.md)
-* [VPP with Kata](./use-cases/using-vpp-and-kata.md)
-* [SPDK vhost-user with Kata](./use-cases/using-SPDK-vhostuser-and-kata.md)
+* [GPU Passthrough with Kata Containers](./use-cases/GPU-passthrough-and-Kata.md): Kata Containers supports passing certain GPUs from the host into the container.
+* [OpenStack\* Zun with Kata Containers](./use-cases/zun_kata.md): How to get Kata Containers to work with OpenStack Zun using DevStack on Ubuntu 16.04.
+* [SR-IOV with Kata Containers](./use-cases/using-SRIOV-and-kata.md): Setup to use SRIOV with Kata Containes and Docker.
```

Comment:
s/Containes/Containers

Reply:
Yep. Note that you can run the [spell checker](https://github.com/kata-containers/tests/tree/master/cmd/check-spelling) we used in the CI manually:

```sh
$ dir=~/go/src/github.com/kata-containers/
$ mkdir -p "$dir" && cd "$dir"
$ git clone https://github.com/kata-containers/tests
$ git clone https://github.com/kata-containers/documentation
$ cd documentation
$ ../tests/cmd/check-spelling/kata-spell-check.sh check ./README.md
```